### DISCUSSION TITLE
想转职不当绳匠了，有什么好工作推荐？

### DISCUSSION THREAD

**Discussion by UranusV** [share121/inter-knot#2440] [其他]
![bangboo](https://github.com/user-attachments/assets/743ad278-9845-47bf-92d4-3dd067918fb0)
这个月基本没接上几单，接到的也都是些脏活累活，报酬还低的要命。月底一算，挣的丁尼还不够邦布的维护修理费，着实是贴钱打工。这世上轻松又赚钱的好工作到底是谁在做啊？

**Comment by xinyuanyoo** [comment]
戴头套当喵吉长官
<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Reply by UranusV** [reply]
喵吉长官才不是头套人，喵吉长官本来就是长那样的！

**Comment by Enthalpy03** [comment]
听说六分街有俩麦片兄妹在招人，丁尼报酬给得很多，不过至于他们招人干嘛我就不清楚了（）

**Reply by UranusV** [reply]
麦片啊，感觉一不小心就进局子了（虽说当绳匠也半斤八两。

**Comment by misaka-lastorder** [comment]
开录像店

**Reply by UranusV** [reply]
你说的是正经录像店吧，是的吧？

**Comment by whatisyoupassword** [comment]
健康最重要了,我兄弟一直在空洞里揽活,上回不走运撞上一大群声骸,差点小命丢了

**Reply by UranusV** [reply]
这是实话。说到底，要不是为了几个丁尼谁愿意整天往空洞那种折寿地方跑、

**Comment by JobNumber-9527** [comment]
可以马路上找辆卡车转生异世界(dog)

**Reply by UranusV** [reply]
只能重开了么，太悲哀了。虽然现在的生活很绝望但是我还是想挣扎一下的啊！

**Comment by MrDucker** [comment]
转行修邦布<img src="https://bbs-static.miyoushe.com/static/2024/03/21/3d1687e289addc7611eba2c8585f4a9f_5782304423493346054.png" alt="丽都漫步-安比" width="75" >

**Reply by UranusV** [reply]
兄弟，你启发了我。我想或许我可以转行当邦布

**Comment by aqswzxgao** [comment]
飞马快递考虑一下<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by YuTinMIn** [comment]
建议隔壁去当旅行者或者开拓者<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
